runtime/compiler/compile/J9Compilation.cpp

```diff
@@ -1526,33 +1526,33 @@ J9::Compilation::notYetRunMeansCold()
                              self()->getOptions()->getInitialBCount() :
                              self()->getOptions()->getInitialCount();
 
-    switch (currentMethod->getRecognizedMethod())
-       {
-       case TR::com_ibm_jit_DecimalFormatHelper_formatAsDouble:
-       case TR::com_ibm_jit_DecimalFormatHelper_formatAsFloat:
-          initialCount = 0;
-          break;
-       default:
-          break;
-       }
-
-    if (currentMethod->containingClass() == self()->getStringClassPointer())
-       {
-       if (currentMethod->isConstructor())
-          {
-          char *sig = currentMethod->signatureChars();
-          if (!strncmp(sig, "([CIIII)", 8) ||
-              !strncmp(sig, "([CIICII)", 9) ||
-              !strncmp(sig, "(II[C)", 6))
-             initialCount = 0;
-          }
-       else
-          {
-          char *sig = "isRepeatedCharCacheHit";
-          if (strncmp(currentMethod->nameChars(), sig, strlen(sig)) == 0)
-             initialCount = 0;
-          }
-       }
+   switch (currentMethod->getRecognizedMethod())
+      {
+      case TR::com_ibm_jit_DecimalFormatHelper_formatAsDouble:
+      case TR::com_ibm_jit_DecimalFormatHelper_formatAsFloat:
+         initialCount = 0;
+         break;
+      default:
+         break;
+      }
+
+   if (currentMethod->containingClass() == self()->getStringClassPointer())
+      {
+      if (currentMethod->isConstructor())
+         {
+         const char *sig = currentMethod->signatureChars();
+         if (!strncmp(sig, "([CIIII)", 8) ||
+            !strncmp(sig, "([CIICII)", 9) ||
+            !strncmp(sig, "(II[C)", 6))
```

Comment:
This is a bit of a nit, but as you're fixing the indentation here, I think it's better to have the operands of the `||` line up with one another than to have the second operand be in an earlier column than the first.  May I ask you to change this to the following?

```
         if (!strncmp(sig, "([CIIII)", 8) ||
             !strncmp(sig, "([CIICII)", 9) ||
             !strncmp(sig, "(II[C)", 6))
            initialCount = 0;
```